docs/user-guide/expressions/aggregation.md

```diff
@@ -17,8 +17,7 @@ Per GROUP `"first_name"` we
 <!-- dprint-ignore-start -->
 
 - count the number of rows in the group:
-    - short form: `pl.count("party")`
-    - full form: `pl.col("party").count()`
+    - full form: `pl.len()`
```

Comment:
Given that this is now the _only_ form given, doesn't seem to be a need to label it "full form" 🤔

Reply:
Removed the "full form" sub line